src/services/api.ts

```diff
@@ -1,4 +1,5 @@
 import axios from 'axios'
+import { HL7File } from '../types';
 
 // TODO config for url
 const baseURL = 'http://localhost:4040/api/users'
```

Comment:
Hey Grant I think, keyword THINK, you made the same mistake of hardcoding the incorrect
url here
Should be const baseURL = 'http://localhost:4040